daprdocs/content/en/reference/resource-specs/configuration-schema.md

```diff
@@ -36,8 +36,15 @@ spec:
         labels:
           - name: <LABEL-NAME>
             regex: {}
+    latencyDistributionBuckets:
+      - <BUCKET-VALUE-MS-0>
```

Comment:
Per line or can be comma separated?

Reply:
it can be comma separated If wrapped in square brackets, was just following the other usage like path matching list